doc/test_in_no_std.md

```diff
@@ -0,0 +1,100 @@
+### test_in_no_std
+
+
+
```

Comment:
Nit multiple blank lines when one would be fine.

Reply:
Will remove